src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+        for (Ball ball : balls) {
+            ball = lottery.getRandomBall();
+            System.out.println(ball.toString());
```

Comment:
Calling `toString()` is redundant here because it is implicitly called when you print an object. You can simply pass the `ball` object to `System.out.println` method.